.github/workflows/ci.yml

```diff
@@ -62,7 +87,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  docs:
+  deprecated_docs:
```

Comment:
Maybe keep this as `docs` until we can get Ashwin to either change the name of the required step, or make it optional?